part-3/pk-sim-compounds-definition-and-work-flow.md

```diff
@@ -146,21 +145,21 @@ After definition of the required parameters the specific clearance or kinetic ra
+PK-Sim® calculates the transcellular specific permeability of the intestinal wall is from the physico-chemical properties of a compound.
 
-In addition to the calculated specific intestinal permeability, experimentally determined permeabilities, e.g. from Caco-2-cell permeability assays can be used. However, due to the large inter-laboratory variability in Caco-2 permeations, a proper calibration of the measured in vitro values and the calculated in silico permeabilities for a defined set of compounds is necessary. If experimentally determined values for intestinal permeabilities are available and the customized calibration method has been implemented in PK-Sim®, this option is then available in the drop-down menu in the _Calculation methods_ window.
+In addition to the calculated specific intestinal permeability, experimentally determined permeabilities, e.g., from Caco-2-cell permeability assays, can be used. However, due to the large inter-laboratory variability in Caco-2 permeations, a proper calibration of the measured in vitro values and the calculated in silico permeabilities for a defined set of compounds is necessary. If the experimentally determined values for the intestinal permeabilities are available and the customized calibration method has been implemented in PK-Sim®, this option is then available in the drop-down menu in the _Calculation methods_ window.
 
 **Specific Intestinal Permeability**
 
-Similarly, the specific intestinal permeability, i.e. the surface area-normalized transcellular permeability of the innermost layer of the intestinal wall, is calculated from the drugs´ lipophilicity and effective molecular weight. The paracellular pathway has been shown to have no impact on the accuracy of prediction of the fraction dose absorbed in humans \[[79](../references.md#79)] and is therefore not accounted for, i.e. the value for the paracellular specific permeability is not automatically calculated. However, the paracellular pathway can be included in the simulation, if desired. You will find the parameter **Intestinal permeability (paracellular)** in the simulation within the parameter group **Permeability**.
+Similarly, the specific intestinal permeability, i.e., the surface area-normalized transcellular permeability of the innermost layer of the intestinal wall, is calculated from the drugs´ lipophilicity and effective molecular weight. The paracellular pathway has been shown to have no impact on the accuracy of prediction of the fraction dose absorbed in humans [[79](../references.md#79)] and is therefore not accounted for, i.e., the value for the paracellular specific permeability is not automatically calculated. However, the paracellular pathway can be included in the simulation, if desired. You will find the parameter **Intestinal permeability (paracellular)** in the simulation within the parameter group **Permeability**.
```

Comment:
@Yuri05  This section is confusing. Above, the calculation of the specifc intestinal permeability is described. Then comes the next section that again talks about the specific intestinal permeability??

Reply:
I believe these two sections must be combined. There is also some duplication of information.

Reply:
yeah. Let's create a new issue and keep it AS IS for now. When modifying this chapter (later) we should also include the info how CACO/PAMPA-calibration is numerically integrated into the model and that the calibration coefficients are accessible to the user via UI.

Reply:
https://github.com/Open-Systems-Pharmacology/docs/issues/271